xml/art_solution_monitoring.xml

```diff
@@ -0,0 +1,251 @@
+      Collects metrics to easily monitor cluster status and health.
+    </para>
+    <formalpara>
+      <title>Link:</title>
+      <para>
+        See <link xlink:href="https://github.com/ClusterLabs/ha_cluster_exporter"/>.
+      </para>
+    </formalpara>
+    <itemizedlist>
+      <title>Export Metrics in Prometheus Format</title>
+      <listitem>
+        <para>
+          Pacemaker cluster summary, nodes, and resource status
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          Corosync ring errors and quorum votes.
+          Currently, only Corosync version&nbsp;2 is supported.
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          SBD devices health status
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          DRBD resources and connections status.
+          Currently, only DRBD version&nbsp;9 is supported.
+        </para>
+      </listitem>
+    </itemizedlist>
+  </sect1>
+
+  <sect1 xml:id="sec-sol-monit-sap-host-exp">
+    <title>&sap; Host Exporter</title>
+    <para>
+      Enables the monitoring of &netweaver;, &hana;, and other applications.
+      The gathered metrics are the data that can be obtained by running the
+      <command>sapcontrol</command> command.
+    </para>
+    <formalpara>
+      <title>Link:</title>
+      <para>
+        See <link xlink:href="https://github.com/SUSE/sap_host_exporter"/>.
+      </para>
+    </formalpara>
+    <itemizedlist>
+      <title>Export Metrics (for &s4h;, &netweaver;, or &hana; hosts) in Prometheus Format</title>
+      <listitem>
+        <para>
+          &sap; start service process list
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          &sap; enqueue server metrics
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          &sap; application server dispatcher metrics
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          &sap; internal alerts
+        </para>
+      </listitem>
+    </itemizedlist>
+  </sect1>
+
+  <sect1 xml:id="sec-sol-monit-more">
+    <title>For More Information</title>
+    <remark>toms 2020-06-10: Need to be reworked which one is useful and should be included</remark>
+    <itemizedlist>
+      <listitem>
+        <para>
+          <link xlink:href="https://documentation.suse.com/suma/4.0/"
+            >&suma;</link>
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          <link xlink:href="https://grafana.com/grafana/dashboards?orderBy=name&amp;direction=asc"
+            >Grafana Dashboards</link>
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          Look also for the some <filename class="extension">.md</filename> files
+          that are included in RPM packages. They contain documentation from the
+          upstream source which could be helpful in isolated data centers without
+          Internet connection.
```

Comment:
```suggestion
          Some <filename class="extension">.md</filename> files
          are included in RPM packages. They contain documentation from 
          upstream sources. This can be helpful in isolated data centers without
          Internet connection.
```